chat/src/main/kotlin/rys/ajaxpetproject/chat/application/service/MessageServiceKludge.kt

```diff
@@ -0,0 +1,63 @@
+package rys.ajaxpetproject.chat.application.service
+
+import org.springframework.stereotype.Service
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import rys.ajaxpetproject.chat.application.port.out.MessageServiceOutPort
+import rys.ajaxpetproject.chat.domain.Message
+import rys.ajaxpetproject.model.MongoMessage
```

Comment:
`MongoMessage` should be located in `*.infrastructure.*` package

Reply:
relocated